src/test/java/nearsoft/academy/bigdata/recommendation/MovieRecommenderTest.java

```diff
@@ -15,7 +15,9 @@
     public void testDataInfo() throws IOException, TasteException {
         //download movies.txt.gz from 
         //    http://snap.stanford.edu/data/web-Movies.html
-        MovieRecommender recommender = new MovieRecommender("/path/to/movies.txt.gz");
+
```

Comment:
You are allowed to change how you read the recommender's data, but you are not allowed to add or make any other changes in the test file. Please remove `recommender.convertFileToCSV();` and make sure to make the required changes so that your class still passes all tests.

Reply:
Fixed, I removed it from the test file and modified the MovieRecommender, it still passed all the tests.